cmd/crane/cmd/validate.go

```diff
@@ -33,7 +32,7 @@ func NewCmdValidate(options *[]crane.Option) *cobra.Command {
 		Use:   "validate",
 		Short: "Validate that an image is well-formed",
 		Args:  cobra.ExactArgs(0),
-		Run: func(_ *cobra.Command, args []string) {
+		RunE: func(_ *cobra.Command, args []string) error {
```

Comment:
I think I want to print FAIL for all errors but exit 1 if any of them fail: maybe collect which formats failed and return that as an error at the end of the loop?

Reply:
Can we do this in another PR? I was a bit surprised that this accepted both `--tarball` and `--remote` in the same invocation.